waddrmgr/manager_test.go

```diff
@@ -1028,28 +1028,32 @@ func testMarkUsed(tc *testContext) bool {
 	addrHash := expectedAddr1.addressHash
 	addr, err := btcutil.NewAddressPubKeyHash(addrHash, chainParams)
 
+	maddr, err := tc.manager.Address(addr)
+	if err != nil {
+		tc.t.Errorf("%s: unexpected error: %v", prefix, err)
```

Comment:
Can't continue with the rest of this test, either return here of make this fatal